docs/quickstarts/cosign-akv-azure.md

```diff
@@ -0,0 +1,441 @@
+
+In this article:
+
+* Sign an image in Azure Container Registry (ACR) with Cosign signatures and keys in AKV
+* Set up an AKS cluster without Azure policy enabled
+* Install OPA Gatekeeper and Ratify
+* Set up Ratify for validating Cosign signatures using keys in AKV
+* Run signed and un-signed images to confirm only signed images are allowed for deployment
+
+> For AKS clusters with Azure policy enabled, Image integrity policy is recommended for ensuring only signed images are deployed
+
+### Prerequisite
+
+The instructions provided in this article are tailored for Linux-based operating systems.
+
+* Install [Cosign](https://docs.sigstore.dev/system_config/installation/)
+* Install and configure the latest [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+* Create or use an [ACR](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli) for storing container images and signatures
+* Create or use an [AKV](https://learn.microsoft.com/en-us/azure/key-vault/general/quick-create-cli) for managing keys
+* Create or using an existing AKS cluster
+ 
+## Sign an image in ACR with Cosign signatures and keys in AKV
+
+1. Configure environment variables.
+
+    ```bash
+    export SUB_ID=<your subscription id>
+
+    export AKV_NAME=<your AKV name>
+    export KEY_NAME=<your key name>
+    export ACR_NAME=<your ACR name>
+    export AKS_NAME=<your AKS name>
+
+    export ACR_RG=<your ACR resource group>
+    export AKV_RG=<your AKV resource group>
+    export AKS_RG=<your AKS resource group>
+    
+    export IMAGE_SIGNED="$ACR_NAME.azurecr.io/net-monitor:v1"
+    export IMAGE_UNSIGNED="$ACR_NAME.azurecr.io/net-watcher:v1"
+
+    export IMAGE_SIGNED_SOURCE=https://github.com/wabbit-networks/net-monitor.git#main
+    export IMAGE_UNSIGNED_SOURCE=https://github.com/wabbit-networks/net-watcher.git#main
+    ```
+
+1. Sign in with Azure CLI
+
+    ```bash
+    az login
+    ```
+
+    To learn more about Azure CLI and how to sign in with it, see [Sign in with Azure CLI](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli).
+
+1. Create a key in your AKV
+
+    Ensure the logged-in identity assigned role `Key Vault Crypto Officer` for creating a key in AKV.
+
+    ```bash
+    az keyvault key create --vault-name $AKV_NAME -n $KEY_NAME --protection software
+    ```
+
+    Get the key id and retrieve the key version:
+
+    ```bash
+    az keyvault key show --name $KEY_NAME --vault-name $AKV_NAME --query "key.kid"
+    ```
+
+    An example output:
+
+    ```text
+    https://<your akv name>.vault.azure.net/keys/<your key name>/<version>
+    ```
+
+    Configure an environment variable for the version:
+
+    ```bash
+    export KEY_VER=<version>
+    ```
+
+1. Build two images in your ACR
```

Comment:
1. Create a key in your AKV , should build the image be "2"

Reply:
If you use `1` always, markdown will render the order for you, which is convenient, since you do not need to calculate the order number.